package.json

```diff
@@ -183,23 +182,19 @@
     "danger-plugin-no-test-shortcuts": "^2.0.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
-    "eslint": "^5.16.0",
+    "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.9.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-config-standard-jsx": "^8.1.0",
     "eslint-config-standard-react": "^9.0.0",
     "eslint-plugin-chai-friendly": "^0.5.0",
     "eslint-plugin-cypress": "^2.8.1",
     "eslint-plugin-import": "^2.19.1",
     "eslint-plugin-jsdoc": "^18.6.0",
     "eslint-plugin-mocha": "^6.2.2",
     "eslint-plugin-no-extension-in-require": "^0.2.0",
-    "eslint-plugin-node": "^11.0.0",
```

Comment:
Was required as peer of `eslint-config-standard`

Reply:
I'm seeing this message in the logs from `npm install` and I'm thinking we should restore this.

```
npm WARN eslint-config-react-app@5.2.0 requires a peer of eslint-plugin-react-hooks@1.x but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-standard@14.1.0 requires a peer of eslint-plugin-node@>=9.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-standard@14.1.0 requires a peer of eslint-plugin-standard@>=4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN rollup-plugin-typescript2@0.25.2 requires a peer of rollup@>=1.26.3 but none is installed. You must install peer dependencies yourself.
```